tests/chains/ethereum/contract/migrations/3_initialize_contract.js

```diff
@@ -0,0 +1,27 @@
+const IBCHandler = artifacts.require("OwnableIBCHandler");
+const IBFT2Client = artifacts.require("IBFT2Client");
+const MockClient = artifacts.require("MockClient");
+const ICS20TransferBank = artifacts.require("ICS20TransferBank");
+const ICS20Bank = artifacts.require("ICS20Bank");
+
+const PortTransfer = "transfer"
+const BesuIBFT2ClientType = "hyperledger-besu-ibft2"
+const MockClientType = "mock-client"
+
+module.exports = async function (deployer) {
+  const ibcHandler = await IBCHandler.deployed();
+  const ics20Bank = await ICS20Bank.deployed();
+
+  for(const promise of [
+    () => ibcHandler.bindPort(PortTransfer, ICS20TransferBank.address),
+    () => ibcHandler.registerClient(BesuIBFT2ClientType, IBFT2Client.address),
```

Comment:
If you use Ganache, you can remove this registration for IBFT2Client.

Reply:
https://github.com/datachainlab/yui-relayer-build/pull/5/commits/892b9d00ebb882dc9d534d6c9654837da82afa19
Removed IBFT2Client